components/experimental/src/lib.rs

```diff
@@ -91,6 +91,7 @@ pub mod provider {
         super::compactdecimal::provider::ShortCompactDecimalFormatDataV1Marker::INFO,
         super::dimension::provider::currency::CurrencyEssentialsV1Marker::INFO,
         super::dimension::provider::percent::PercentEssentialsV1Marker::INFO,
+        super::dimension::provider::units::UnitsDisplayNameV1Marker::INFO,
```

Comment:
issue: you need to wire up the impl with `impl_units_display_name_v1_marker!(Baked)`

Reply:
Sure, I added it in #5000